source/_posts/2015-08-06-open-sourcing-energy.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "Open Sourcing Energy"
+date: 2015-08-06 13:54
+comments: false
+categories: [ios, mobile, energy, open source, oss]
+author: Orta Therox
+github-url: https://www.github.com/orta
+twitter-url: http://twitter.com/orta
+blog-url: http://orta.io
+
+---
+
+The Artsy Mobile team is pretty agressive in our stance on [Open Source by Default](http://code.dblock.org/2015/02/09/becoming-open-source-by-default.html). We've talked about it at [conferences](https://www.youtube.com/watch?v=2DvDeEZ0NDw&spfreload=10) [around](https://www.youtube.com/watch?v=SjjvnrqDjpM) [the](https://www.youtube.com/watch?v=zPbLYWmLPow) [world](https://speakerdeck.com/orta/ios-at-artsy), in [renowned magazines](www.objc.io/issues/22-scale/artsy) and on [our blog](http://artsy.github.io/blog/2015/04/28/how-we-open-sourced-eigen/).
+
+It's worth mentioning that we don't just talk externally about Open Source. Internally the entire development team runs talks, explains how GitHub works and what the implications of all our work in the open are. We strive for an open culture in more than just the development team. The development team is just further along in the process.
+
+The Open Source app idea started with an experiment in the Summer of 2014, asking "What does a truely Open Source App look like?"  The outcome of that was our Swift Kiosk app, [Eidolon](https://github.com/artsy/eidolon/). Open from day one. We took the knowledge from that, and applied it to our public facing app, [Eigen](https://github.com/artsy/eigen/). Open from day 806. That made 2/3rds of our apps Open Source. 
+
+Let's talk about our final app, [Energy](https://github.com/artsy/energy). Open from day 1433 and ~3500 commits.
+
+<!-- more -->
+
+----------------
+
+![ENERGY](/images/2015-08-01-open-sourcing-energy/ENERGY.png)
+
+Energy is commonly known as [Artsy Folio](http://folio.artsy.net). It's a tool for Artsy's Partners to showcase their artworks on the go, and quickly email them. Here's a beautiful splash showing it in action.
+
+{% expanded_img http://folio.artsy.net/images/cover-bbf6fdf4.jpg Folio overview %}
```

Comment:
Eager contributors may note that Energy on iPhone doesn't support the landscape rotation...